TLYShyNavBar/TLYShyViewController.m

```diff
@@ -61,8 +61,12 @@ - (CGFloat)totalHeight
 
 // This method is courtesy of GTScrollNavigationBar
 // https://github.com/luugiathuy/GTScrollNavigationBar
-- (void)_updateSubviewsToAlpha:(CGFloat)alpha
+- (void)_updateViewsToAlpha:(CGFloat)alpha
```

Comment:
should probably rename `_updateViewsToAlpha` to `_updateViewsToAlphaIfNeeded`. Then, call it directly, without any if statements. Within the method, we add a `switch`, and apply the changes.